### PR TITLE
Update user guide - user profile

### LINKED ISSUE #55: Update User guide

### LINKED ISSUE #63: Update profile commands in the user guide for v1.1

### PR DESCRIPTION
**Pull request by orzymandias**
#55

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -24,9 +24,10 @@ title: User Guide
...
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[3.2.4. Viewing an internship application: `view -int`](#Viewing-an-internship-application) <br />
 &nbsp;&nbsp;&nbsp;&nbsp;[3.3. User Profile](#user-profile) <br />
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[3.3.1. Adding skills and achievement: `add -me`](#Adding-skills-and-achievement) <br />
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[3.3.2. Deleting skills and achievement: `delete -me`](#Deleting-skills-and-achievement) <br />
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[3.3.3. Editing skills and achievement: `view -me`](#Editing-skills-and-achievement) <br />
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[3.3.1. Adding to user profile: `add -me`](#Adding-to-user-profile) <br />
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[3.3.2. Deleting from user profile: `delete -me`](#Deleting-from-user-profile) <br />
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[3.3.3. Editing user profile: `view -me`](#Editing-user-profile) <br />
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[3.3.4. Viewing item in user profile: `view -int`](#Viewing-item-in-user-profile) <br />
```
Comment (ZoroarkDarkrai) [inline docs/UserGuide.md]: @seanjyjy is going to redo this part(?) so should I ignore this?

Comment (orzymandias) [reply docs/UserGuide.md]: You can ignore this first. The content page can be changed after all the headers are in.

Comment (orzymandias) [reply docs/UserGuide.md]: nvm, its good to merge, I have updated the contents page to link to my headers.

**Review by ZoroarkDarkrai — APPROVED** [review]
Lgtm!

**Pull request merged**